openlibrary/i18n/es/messages.po

```diff
@@ -1664,7 +1664,7 @@ msgstr "Libro aleatorio"
 
 #: nav_head.html:15 nav_head.html:93
 msgid "Recent Community Edits"
-msgstr "Recientes ediciones de la comunidad"
+msgstr "Ediciones recientes de la comunidad"
```

Comment:
The changes in this file appear unrelated to the pull request. Please put them in a separate  PR.

Reply:
Reverted changes not related to the PR. Thank you.

Reply:
@tfmorris, I'm not sure how this is done:
- Squash the typo cleanup commit into the main commit
- Rebase onto the current master to remove the merge commit

Reply:
Thanks for updating the PR. I'll take care of the rebasing it, but for the future here's how it's done. First we figure out how many commits we need to modify (in this case 7) and do an interactive rebase of them all.

`git rebase -i HEAD~7`

which gives you an editor session that looks like this:

```
pick eb8ab51 Made footer's HTML translatable
pick a18d382 Fixed some typos
pick 76b9883 Made footer's HTML translatable: added the missing translatable strings
pick 73c78b7 Fix git hash version i18n
pick 377e121 Added a missing translatable string
pick caf9507 Reverted unrelated changes to the PR
pick 23961be Clean up trailing whitespace

# Rebase ef7d551..23961be onto ef7d551 (7 commands)
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
#
# These lines can be re-ordered; they are executed from top to bottom.
```
we can edit the buffer so that it looks like this:

```
pick eb8ab51 Made footer's HTML translatable
f 23961be Clean up trailing whitespace
f 76b9883 Made footer's HTML translatable: added the missing translatable strings
f 377e121 Added a missing translatable string
d a18d382 Fixed some typos
d caf9507 Reverted unrelated changes to the PR
pick 73c78b7 Fix git hash version i18n
```

which pulls all your commits to the template (plus one that I made to remove some trailing whitespace) together and merges them into a single commit. The two commits to the messages.po file we just drop since they counter each other.

Then to update the branch to the current master (and get rid of the merge commit) we do:

`git rebase master`

That leaves us with a branch containing just two commits which can be cleanly merged into the current master with no merge conflicts.